ethcore/types/src/header.rs

```diff
@@ -24,6 +24,8 @@ use bytes::Bytes;
 use rlp::{Rlp, RlpStream, Encodable, DecoderError, Decodable};
 use BlockNumber;
 
+use machine::WithMetadataHeader;
```

Comment:
I'm not sure how to import the `WithMetaDataHeader` trait here without creating a cyclical dependency.

Reply:
Huh.  A lot of the codebase has been refactored since block metadata was removed.  Looking for a better place for the definition of the `WithMetadataHeader` trait.